tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -58,8 +57,7 @@ class TGradientDescent
 
 private:
    size_t   fBatchSize; ///< Batch size to use for the training.
-   size_t   fStepCount; ///< Number of steps performed in the current
-   ///< training sessiong.
+   size_t fStepCount;   ///< Number of steps performed in the current training session
```

Comment:
You're welcome to overrule clang-format; this is one of the known limitations. And yes, spelling fixes are always welcome :-)